src/data/markdown/docs/30 xk6-browser/01 Getting started/02 Installation.md

```diff
@@ -0,0 +1,36 @@
+---
+title: 'Installation'
+excerpt: 'A step-by-step guide on how to install xk6-browser.'
+---
+
+xk6-browser is currently being developed as a [k6 extension](/extensions). You have to run a k6 version built with the browser extension to use the [browser-level APIs](#browser-level-apis) in your k6 tests.
```

Comment:
This link (`browser-level APIs`) doesn't work. I think it would be good to note here that the instructions on how to build a k6 version with the browser extension are below (or just link to it). People who are curious will click on it, and people who don't care about the details can ignore it.

Reply:
I'll just move the instructions below after this paragraph :)